.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -8,6 +8,11 @@ on:
         required: true
         default: 'develop'
         type: string
+      delete-bee:
```

Comment:
Useful to leave the bee around for debugging test issues.

![image](https://github.com/broadinstitute/rawls/assets/484484/e0a36e08-aa63-4c51-8bca-bab9a5a1cc7f)